lib/linguist/languages.yml

```diff
@@ -2170,6 +2163,13 @@ KiCad Layout:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 187
+KiCad Legacy Layout:
+  type: data
+  extensions:
+  - ".brd"
+  tm_scope: source.pcb.board
+  ace_mode: text
+  language_id: 140848857
 KiCad Schematic:
```

Comment:
Are schematics still in use by KiCad projects? From what I saw, they featured a very similar syntax to the legacy `.brd` files.

Reply:
Are you referring to `.sch` files? 
Yes, those are not deprecated

Reply:
The plan is to switch to an s-expression based syntax similar to the one used in the new KiCad layout in KiCad 5 though (but I think the plan is to change the file extension as well when doing this).